internal/dataexchange/dxfactory/factory.go

```diff
@@ -27,15 +27,23 @@ import (
 )
 
 var (
-	OldFFDXPluginName = "https"
 	NewFFDXPluginName = (*ffdx.FFDX)(nil).Name()
 )
 
 var pluginsByName = map[string]func() dataexchange.Plugin{
 	NewFFDXPluginName: func() dataexchange.Plugin { return &ffdx.FFDX{} },
 }
 
-func InitConfig(config config.Section) {
+func InitConfig(config config.ArraySection) {
+	config.AddKnownKey(dataexchange.DataExchangeConfigType, "ffdx")
```

Comment:
Could use `NewFFDXPluginName` here (which probably doesn't need to be named "new" anymore)